MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Reflectometry/QtReflSettingsTabView.h

```diff
@@ -0,0 +1,64 @@
+#ifndef MANTID_CUSTOMINTERFACES_QTREFLSETTINGSTABVIEW_H_
+#define MANTID_CUSTOMINTERFACES_QTREFLSETTINGSTABVIEW_H_
+
+#include "MantidQtCustomInterfaces/Reflectometry/IReflSettingsTabView.h"
+#include "ui_ReflSettingsTabWidget.h"
+
+namespace MantidQt {
+namespace CustomInterfaces {
+
+// Forward decs
+class IReflSettingsTabPresenter;
+
+/** QtReflSettingsTabView : Provides an interface for the "Settings" tab in the
+Reflectometry (Polref) interface.
+
+Copyright &copy; 2016 ISIS Rutherford Appleton Laboratory, NScD Oak Ridge
+National Laboratory & European Spallation Source
+
+This file is part of Mantid.
+
+Mantid is free software; you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation; either version 3 of the License, or
+(at your option) any later version.
+
+Mantid is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+File change history is stored at: <https://github.com/mantidproject/mantid>
+Code Documentation is available at: <http://doxygen.mantidproject.org>
+*/
+class QtReflSettingsTabView : public QWidget, public IReflSettingsTabView {
+  Q_OBJECT
+public:
+  /// Constructor
+  QtReflSettingsTabView(QWidget *parent = 0);
+  /// Destructor
+  ~QtReflSettingsTabView() override;
+  /// Returns the presenter managing this view
+  IReflSettingsTabPresenter *getPresenter() const;
```

Comment:
This is OK, but I started wondering if we ought to return a `WeakPtr`. I haven't figured that out properly yet.

In any case at least a ptr can be assigned and checked for a `nullptr`